package.json

```diff
@@ -14,8 +14,8 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "pretest": "jshint .",
-    "test": "mocha -R spec --timeout 60000"
+    "test": "mocha -R spec --timeout 60000",
+    "posttest": "eslint --ignore-path .gitignore ."
```

Comment:
Agreed. Doing linting first is way better for CI speed - no need to wait 30 minutes just to find out that even though _all_ the tests passed, you have a trailing whitespace somewhere.

Also, all of our other repos/packages do their linting in `pretest`.